content/en/docs/concepts/overview/components.md

```diff
@@ -18,7 +18,7 @@ deliver a functioning Kubernetes cluster.
 ## Master Components
 
 Master components provide the cluster's control plane. Master components make global decisions about the
-cluster (for example, scheduling), and they detect and respond to cluster events (for example, starting up a new pod when a replication controller's `replicas` field is unsatisfied).
+cluster (for example, scheduling), and they detect and respond to cluster events (for example, starting up a new [pod](/docs/concepts/workloads/pods/pod) when a replication controller's `replicas` field is unsatisfied).
```

Comment:
Would it make sense to use a glossary tooltip here? It works as a link and also gets a tooltip for cursor-based GUIs.

```suggestion
cluster (for example, scheduling), and they detect and respond to cluster events (for example, starting up a new {{< glossary_tooltip text="Pod" term_id="pod" >}} when a replication controller's `replicas` field is unsatisfied).
```

(an aside: this should probably talk about “Deployment” rather than “replication controller” as well).

Reply:
tooltip can work I guess. 
Did you mean the text should read like 

```
 when a Deployment's `replicas` field is unsatisfied).
``` 
?

Reply:
@hpandeycodeit that's what I had in mind.